frontend/app/framework/angular/forms/editors/date-time-editor.component.ts

```diff
@@ -198,4 +225,44 @@ export class DateTimeEditorComponent extends StatefulControlComponent<{}, string
 
         this.suppressEvents = false;
     }
+
+    public setLocalMode(isLocalMode: boolean) {
+        this.isLocalMode = isLocalMode;
+
+        if (!this.dateControl.value) {
+            return null;
+        }
+
+        let parsed: DateTime | null;
+
+        if (this.timeControl.value) {
+            const combined = `${this.dateControl.value}T${this.timeControl.value}`;
+            parsed = DateTime.tryParseISO(combined, this.isLocalMode);
+        } else {
+            parsed = DateTime.tryParseISO(`${this.dateControl.value}T00:00:00`, this.isLocalMode);
+        }
+
+        if (parsed) {
+            let dateString: string;
+
+            if (this.isLocalMode) {
+                dateString = parsed.toStringFormat('yyyy-MM-dd');
```

Comment:
A empty lines would make it easier to understand, e.g. after timeControl. and before dateControl.